Source/Core/Core/HW/WII_IPC.cpp

```diff
@@ -170,6 +229,196 @@ void Shutdown()
 {
 }
 
+static std::string GetAVERegisterName(u8 address)
```

Comment:
Could this function use a switch statement instead?

Also, maybe return a `std::string_view` ?

Reply:
A switch statement would be a problem because of these lines:

https://github.com/dolphin-emu/dolphin/blob/b551c8b8cb4bb511a8c698c0c2adbe8ae7aeea93/Source/Core/Core/HW/WII_IPC.cpp#L250-L253

I think `std::string_view` would be fine though since the string constants live forever.

Reply:
Oh whoops, didn't look closely enough :D.